### PR TITLE
Messages from GoodStrap to ui-kit

### PR DESCRIPTION
**Pull request by strouhaljan**
JIRA: ONE-4642

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1263/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2858/ : FAILURE in 14m 50s

File: libs/sdk-ui-kit/src/List/InvertableList.tsx [hunk 1]
```diff
@@ -6,12 +6,12 @@ import keyBy from "lodash/keyBy";
...
 import has from "lodash/has";
+import { Dictionary } from "lodash";
 import { guidFor } from "@gooddata/goodstrap/lib/core/Guid";
-import Message from "@gooddata/goodstrap/lib/Messages/Message";
 
-import { MultiSelectList } from "./MultiSelectList";
+import { Message } from "../Messages";
 import { Input } from "../Form";
-import { Dictionary } from "lodash";
+import { MultiSelectList } from "./MultiSelectList";
```
Comment (ivan-nejezchleb) [inline libs/sdk-ui-kit/src/List/InvertableList.tsx:14]: I am surprised that it works. It should import LegacyMultiSelectList (the original one from GS from root folder, not from @next folder)

File: libs/sdk-ui-kit/styles/scss/messages.scss [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+    font-size: 14px;
+    text-align: left;
+
+    a:not(.gd-message-dismiss) {
+        color: inherit;
+        text-decoration: underline;
+    }
+
+    &.progress {
+        color: $gd-color-highlight;
```
Comment (ivan-nejezchleb) [inline libs/sdk-ui-kit/styles/scss/messages.scss:177]: I would expect that this hardcoded primary color (all occurences) would be replaced by something more - customizable ;)

**Review by ivan-nejezchleb — CHANGES_REQUESTED** [review]

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1287/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2879/ : FAILURE in 4m 36s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1294/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2887/ : FAILURE in 3m 27s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1295/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2889/ : FAILURE in 13m 18s

File: libs/sdk-ui-kit/src/List/InvertableList.tsx (hunk 1 above)
Comment (strouhaljan) [reply libs/sdk-ui-kit/src/List/InvertableList.tsx:14]: Just a diff misinterpretation.

File: libs/sdk-ui-kit/styles/scss/messages.scss (hunk 2 above)
Comment (strouhaljan) [reply libs/sdk-ui-kit/styles/scss/messages.scss]: Made customizable.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1296/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2890/ : FAILURE in 13m 22s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1298/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2892/ : SUCCESS in 10m 54s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by strouhaljan** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1301/ : SUCCESS in 24s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2896/ : SUCCESS in 10m 34s

**Review by ivan-nejezchleb — APPROVED** [review]

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/463/ : FAILURE in 2m 05s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/722/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2899/ : SUCCESS in 14m 13s

**Pull request merged**